src/Elmish.WPF.Tests/ViewModelTests.fs

```diff
@@ -818,6 +815,87 @@ module OneWaySeqLazy =
     }
 
 
+  type TestClass (id: int, data: string) =
+    member _.Id = id
+    member _.Data = data
+    override __.GetHashCode() = 0
+    override __.Equals that =
+      // All instances of TestClass are considered equal.
+      // Not very helpful, but a valid implementation.
+      match that with
+      | null -> false
+      | :? TestClass as x -> true
```

Comment:
Actually, you can remove the `null` check. Type checks à la `:? TestClass` will only succeed if it's not null.

Reply:
Sure.  Force pushed that change.